test/glific_web/schema/wa_search_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule GlificWeb.Schema.WaSearchTest do
+  use GlificWeb.ConnCase
+  use Wormwood.GQLCase
+  alias Glific.Seeds.SeedsDev
+
+  setup do
+    default_provider = SeedsDev.seed_providers()
+    SeedsDev.seed_organizations(default_provider)
+    SeedsDev.seed_contacts()
+    SeedsDev.seed_wa_managed_phones()
+    SeedsDev.seed_wa_groups()
+    SeedsDev.seed_wa_messages()
+    :ok
+  end
+
+  load_gql(:wa_search, GlificWeb.Schema, "assets/gql/searches/wa_search.gql")
+
+  test "wa_search for conversations", %{staff: user} do
```

Comment:
need to add few more testcases to cover the results are consistent using different filter and opts